src/components/ProjectBoard/ProjectBoard.js

```diff
@@ -0,0 +1,29 @@
+import { useSelector } from "react-redux";
+import Column from "../Column/Column";
+import Task from "../Task/Task";
+
+function ProjectBoard({ selectedProject }) {
```

Comment:
So overall I see what you're doing and this is great! I wonder if we should really be manipulating GS for the selected project though?   I'm sure I'm the one that originally had projects being passed down as props